plots.py

```diff
@@ -71,8 +74,15 @@ def semilog_cases_since(countries, num_cases=100, time_constant_type=10, num_dat
 
     cases, deaths, recovered = parse_country_data()
 
+    if data_type == "deaths":
+        plot_data = deaths
+    elif data_type == "recovered":
+        plot_data = recovered
+    else:
+        plot_data = cases
```

Comment:
I think the dataframe should be passed to the function instead of the type of data. You could add functions like `semilog_deaths_since` as syntactic sugar instead

Reply:
Good idea